### ISSUE TITLE
Updated itself without warning or consent. All configurations are gone

### ISSUE THREAD

**Issue by Rubiksensei** [SoftFever/OrcaSlicer#6607]
### Is there an existing issue for this problem?

- [X] I have searched the existing issues

### OrcaSlicer Version

2.1.1

### Operating System (OS)

Windows

### OS Version

Windows 10 version 22H2

### Printer

Bambulab P1S

### How to reproduce

Happened on itself. Don't know when or why

### Actual results

Opened the software to the full reinstallation and first configuration process

### Expected results

Opening the software and get back to work like usual

### Project file & Debug log uploads

[log.zip](https://github.com/user-attachments/files/16815471/log.zip)

### Checklist of files to include

- [X] Log file
- [ ] Project file

### Anything else?

In short, opened the software this morning and it seems it update itself without any warning or consent. All configurations are gone it's like fresh new install. Apparently no way to open the previous install to export the config.json. or my filaments/printers profiles I tweaked over time. Not gonna lie, it's really a pain to redo everything
